enhancements/bigquery_row_count.py

```diff
@@ -0,0 +1,50 @@
+import yaml
+from elastalert.enhancements import BaseEnhancement, DropMatchException
+from elastalert import util
+
+class BigQueryRowCount(BaseEnhancement):
+    
+    def process(self, match):
+        if self.check_match(match):
+            with open("/opt/elastalert/config.yaml") as config:
+                conf = yaml.safe_load(config)
+                parsed_conf = util.build_es_conn_config(conf)
+                es_client = util.elasticsearch_client(parsed_conf)
+
+            tmp_table_id = match["gcp"]["audit"]["bigquery"]["tmp_table_id"]
+            row_count = self.search_row_count(tmp_table_id, es_client)
+            es_client.close()
+
+            # Filter out all exports that are smaller than 1000 rows 
+            if int(row_count) < 1000:
+                raise DropMatchException()
+
+            
+    def check_match(self, match):
```

Comment:
For this, could we not simplify it to perhaps a try/catch where we try and get the `tmp_table_id` and if it can't access it through the dict then return False?

Reply:
Yes, I was going to say this is crap!